src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -61,17 +62,24 @@ def __init__(self):
       for config in trials_config:
         if config['app_name'] != app_name:
           continue
-        self.trials[config['app_args']] = config['probability']
+        self.trials[config['app_args']] = {'probability': config['probability']}
```

Comment:
You can just simplify this with

```python
self.trials[config['app_args']] = {
  'probability': config['probability'],
  'contradicts': config.get('contradicts', [])
}
```

Reply:
+1, this is actually required as in the previous code the 'contradicts' field stays uninitialized.

Reply:
done